cmds/db-manager/main.go

```diff
@@ -0,0 +1,219 @@
+// Bootstrap script for Database deployment and migration
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/cockroachdb"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	dssCockroach "github.com/interuss/dss/pkg/cockroach"
+	"go.uber.org/zap"
+)
+
+// MyMigrate is an alias for exstending migrate.Migrate
+type MyMigrate struct {
+	*migrate.Migrate
+}
+
+// Direction is an alias for int indicating the direction and steps of migration
+type Direction int
+
+func (d Direction) String() string {
+	if d > 0 {
+		return "Up"
+	} else if d < 0 {
+		return "Down"
+	}
+	return "No Change"
+}
+
+var (
+	path      = flag.String("migration_dir", "", "path to db migration files directory")
```

Comment:
Assuming we adopt the separate-databases approach, I now suggest that we reduce the possibility of inconsistency (and reduce complexity) by naming migrations folders according to database name, and just pass in the database name as a single argument to this tool.

Reply:
We could do that, but would it be clearer to to have "rid-schema" as the dir name instead of "defaultdb" ? I don't have a strong opinion on this, just wondering if it would be confusing or not. What I'd do to to keep your suggestion would be to pass ./db-schemas/defaultdb to --schema_dir and take the leaf directory name as the db name to upgrade